### PR TITLE
1 sign up

### PR DESCRIPTION
**Pull request by rcvink**
#1

### CODE REVIEW

**Review by MissDove — COMMENTED** [review]
This is really bad code! (joke!)

File: .rspec [hunk 1]
```diff
@@ -1 +1,3 @@
 --require spec_helper
+--format documentation
```
Comment (MissDove) [inline .rspec:2]: good call!

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+require 'bcrypt'
+require 'dm-validations'
+
+class User
+  include DataMapper::Resource
+  include BCrypt
```
Comment (edpe) [inline app/models/user.rb:6]: totally rad

**Review by auberdream — COMMENTED** [review]
~ ☯  a m a z i n g . r a r ☯  ~

File: spec/features/user_account_spec.rb [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+      end
+
+      scenario 'email is provided with incorrect format' do
+        User.create(name: 'Test Bloggs', email: 'invalidemail', username: 'tester', password: 'test123', password_confirmation: 'test123')
+        expect(User.all.count).to eq 0
+      end
+
+    end
+
+    context 'displays error if' do
```
Comment (edpe) [inline spec/features/user_account_spec.rb:68]: b e  y o u r  v a p o r w a v e

**Review by edpe — APPROVED** [review]

**Pull request merged**